packages/web-components/@sfx/search-box/test/unit/search-box.test.ts

```diff
@@ -0,0 +1,239 @@
+import { SEARCHBOX_EVENT } from '../../src/utils';
+import { expect, spy, stub } from '../utils';
+import SearchBox from '../../src/search-box';
+import { TemplateResult, html, LitElement } from 'lit-element';
+
+describe('SearchBox Component', () => {
+  let searchbox;
+  let windowDispatchEvent;
+
+  beforeEach(() => {
+    searchbox = new SearchBox();
+    windowDispatchEvent = spy(window, 'dispatchEvent');
```

Comment:
This should probably be a stub since you don't want it to actually dispatch the event.